### PR TITLE
New package: Venom-0.5.4

### LINKED ISSUE #17786: Package request:Venom
a modern Tox client for the GNU/Linux desktop:

https://github.com/naxuroqa/Venom

### PR DESCRIPTION
**Pull request by ndowens**
Closes #17786

Signed-off-by: Nathan Owens <ndowens04@gmail.com>

### CODE REVIEW

File: srcpkgs/Venom/template [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+ opus-devel libvpx-devel gstreamer1-devel"
+short_desc="Modern Tox client for the desktop"
+maintainer="Nathan Owens <ndowens04@gmail.com>"
+license="GPL-3.0-or-later"
+homepage="://github.com/naxuroqa/Venom/"
+distfiles="https://github.com/naxuroqa/Venom/archive/${version}.tar.gz"
+checksum=2e4bbab0b68a1a38ffcc5f9991fb5d95391c207e931edb2ce6a1e1edf6795b1b
+
+if [ "$CROSS_BUILD" ]; then
+	hostmakedepends+=" glib-devel"
```
Comment (xtraeme) [inline srcpkgs/Venom/template]: just add this to `hostmakedepends`, no conditional.

Comment (ndowens) [reply srcpkgs/Venom/template]: Done

File: srcpkgs/Venom/template [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+build_style=meson
+hostmakedepends="vala pkg-config glib-devel"
+makedepends="gtk+3-devel libgee-devel gspell-devel
+ libcanberra-devel sqlcipher-devel json-glib-devel
+ libsoup-devel toxcore-devel libsodium-devel
+ opus-devel libvpx-devel gstreamer1-devel"
+short_desc="Modern Tox client for the desktop"
+maintainer="Nathan Owens <ndowens04@gmail.com>"
+license="GPL-3.0-or-later"
+homepage="://github.com/naxuroqa/Venom/"
```
Comment (daniel-eys) [inline srcpkgs/Venom/template]: https

Comment (ndowens) [reply srcpkgs/Venom/template]: Done

**Pull request merged**